mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.kt

```diff
@@ -19,6 +20,7 @@ class PassCodeActivity : MifosPassCodeActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        findViewById<NestedScrollView>(R.id.cl_rootview).setBackgroundColor(android.R.attr.colorBackground)
```

Comment:
@Aditya-gupta99 ,
can you convert it to butterknife?

Reply:
Okay @avinash14022002 , I will do that